src/main/java/org/usfirst/frc4904/robot/commands/SetHoodAngle.java

```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.subsystems.Hood;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+public class SetHoodAngle extends CommandBase {
```

Comment:
should it be some sort of motor position constant?

Reply:
yes, absolutely